README.md

```diff
@@ -1,13 +1,18 @@
+Yoshino platform
+==============
 Copyright (C) Sony Mobile Communications 2017
-=============================================
+---------------------------------------------
 
-This is the Android device configuration for yoshino platform.
+Android device configuration for the yoshino platform.
```

Comment:
Don't forget to add the SoC name here (and all other platforms where it was missed - I don't want to comment the same thing on all :wink:).

Reply:
yep